### DISCUSSION TITLE
Null values giving error in Computed columns

### DISCUSSION THREAD

**Discussion by Sandy381** [fbaligand/kibana-enhanced-table#313] [General]
Hi,

I'm trying to generate computed column to one of my columns in table.

But when i try to generate a computed column out of string column it generates the below error.

![Screenshot 2023-03-21 at 11 39 44 AM](https://user-images.githubusercontent.com/74658016/226531712-6dc9698a-04c4-4168-a755-1cbc48e2b25a.png)
![Screenshot 2023-03-21 at 11 39 06 AM](https://user-images.githubusercontent.com/74658016/226531739-590fdb40-de99-413b-a799-4a6b840559b4.png)

How can i Overcome this error?

**Comment by fbaligand** [comment]
How many columns do you have?
If you have 4 columns, last column is col3, because col index is 0-based
